src/redux/reducers/token.js

```diff
@@ -0,0 +1,14 @@
+import {createSlice} from "@reduxjs/toolkit";
+
+const tokenSlice = createSlice({
+  name: "token",
+  initialState: {token: null},
+  reducers: {
+    setToken: (state, action) => {
+      state.token = action.payload;
```

Comment:
A. Here it should be `state = actions.payload;`, Otherwise, Check B

Reply:
i changed it to state = actions.payload now, but there still is no change. the login still doesnt work and there is still nothing happening when i try to login